src/views/AboutMonarch.md

```diff
@@ -1,137 +1,82 @@
-
-##### Analysis
-
-
-<figure class="right" href="#fig4">
-  <img src="../assets/img/pheno_species_coverage.jpg" height="200px">
-  <figcaption>
-
-Fig 4. Availability of curated phenotypes (black) attributed to human genes (direct or inferred via orthology) across a range of model organisms (Hs, Homo sapiens; Mm, Mus musculus; Rn, Rattus norvegius; Dr, Danio rerio; Ce, Ceanorabditis elegans). Merge shows the overall representation of phenotypes to unique human genes, where at least one organism contriubtes phenotypes. Model organisms may account for at least half of the overall phenotypic knowledge to understand human gene function. (Data as of March 2014.)
-
-  </figcaption>
-</figure>
-
-
-Given the integrated data sources, and their semantic mappings, we can start to do broad analysis of the data landscape. For example, we have found that while &lt;40% of human genes have been linked to phenotypes either directly, via GWAS studies, or inferred through disease associations, there is &gt;75% coverage (via orthology) when you expand to the five most-typically studied model organisms ([Fig 4](#fig4)). As we expand our database to include other organisms, this is sure to rise.
-
-We can also confirm what we have known for a long time, which is that different model organisms are used to study different areas of biology.
-
-
-#### Collaborations and Partners
-
-We are proud to work with several organizations to develop ontologies, drive community standards, research rare disease, and influence publishers and the research community to make research results more identifiable and reproducible, including:
+<h2 class="page-title">The Monarch Initiative</h2>
+<div class="row">
 
+<p>The Monarch Initiative is an integrative data and analytic platform connecting phenotypes to genotypes across 
+species, bridging basic and applied research with semantics-based analysis. The correlation of phenotypic outcomes 
+and disease with genetic variation and environmental factors is a core pursuit in biology and biomedicine.</p>
+ 
+</div>
+<div class="row">
+    <div class="col-lg-6">
+        <h5 class="figure-title">The phenotype annotation coverage of human coding genes.</h4>
+        <figure class="cross-species">
+          <img src="../assets/img/cross-species.jpeg" style="max-height:300px;" />
+          <figcaption>
+           Fig 1. This figure shows the availability of curated phenotypes that have been attributed to human genes 
```

Comment:
Update the text for figure one with: 
Model organisms provide key insight into phenotypic manifestations of human coding genes. Human coding genes with disease/phenotype-causing mutations are shown on the left; model organism orthologs of human coding genes with associated phenotypes are on the right. By including just 5 species, we can boost coverage by 64%, up to ~82% of human protein coding genes reported by the HUGO Gene Nomenclature Committee (HGNC).

Reply:
Do we need to call out the OMIM statistic?  I don't think we know why our count is 200 less OMIM, and we may be including genes that they are not.

Reply:
This was part of a discussion with @jmcmurry, @pnrobinson, and @mellybelly about the latest on the numbers in this figure, and we agreed that this legend would be adequate for a grant submission (last week). We don't have to use the entire text on the site. How about we shave as follows:

Model organisms provide key [...] Initiative. The dotted line (left panel) denotes approx. 200 genes filtered out from the total number of genes with a Mendelian phenotype due to a paucity of supporting data for the genotype - to - disease relation in entries from GWAS catalog, CTD, and parts of ClinVar, Orphanet, and OMIM (see https://omim.org/statistics/geneMap).

Reply:
seems fine; I would remove:
> 200 genes filtered out from the total number of genes with a Mendelian phenotype due to a paucity of supporting data for the genotype - to - disease relation in entries from GWAS catalog, CTD, and parts of ClinVar, Orphanet, and OMIM

as this is not accurate

Reply:
Oy. 😬 Well, that removes that whole paragraph and the reason for the dotted line. 
Do you have an idea for the reason why there are only ~3,900 in our collection, but ~4,100 in the OMIM link that @pnrobinson shared? .... effectively, please offer some words to replace the ones you are removing. :bowtie:

Reply:
There are 3463 genes associated with a causal association to a disorder from our omim ingest.  Some possibilities might include:
1. We are differing on what we call a gene, for example, we categorize many omim genes as heritable phenotypic markers (eg https://www.ncbi.nlm.nih.gov/gene/619478)
2. We are more strictly filtering on gene-disease relationship
3. Disease/phenotype/measurement category issues
4. Theres a bug in our ingest code

Reply:
And then, there are these numbers from your message on 2019-07-29: 
- Number of human protein coding genes with a causal gene to disease association: 3937
- Number of human protein coding genes with a causal gene to disease or gene to phenotype association: 4052

Reply:
We could also just update the figure to reflect the OMIM number.

Reply:
I've added this to our Data call topics for this week. :bowtie:

Reply:
Although I am aware that @mellybelly does not like the OMIM number, I also realize that the reasons for removing the ~200 genes are many and difficult to narrow down into a figure legend. 

So, in order to remove the block this PR - I have created a new image and updated legend.  

@iimpulse The new image can be found in the same directory, [here](https://github.com/monarch-initiative/monarch-ui/blob/master/src/assets/img/phenotype-cov-hum-6.png). The file name is phenotype-cov-hum-6.png

The legend can be: ```Model organisms provide key insight into phenotypic manifestations of human coding genes. Human coding genes with disease/phenotype-causing mutations are shown on the left; model organism orthologs of human coding genes with associated phenotypes are on the right. By including just 5 species, we can boost coverage by 61%, up to ~82% of human protein coding genes reported by the HUGO Gene Nomenclature Committee (HGNC). There are 13 sources of data integrated within the Monarch Initiative.``` 

![phenotype-cov-hum-6](https://user-images.githubusercontent.com/1714621/62517403-33210180-b7dc-11e9-9eaf-f5080ef0b261.png)